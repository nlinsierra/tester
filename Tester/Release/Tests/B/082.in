5H 9D 9S QH JC 3C JS 3D TC 8S 9C TD 4D AS 7D 7S 2D KH 8C QD
