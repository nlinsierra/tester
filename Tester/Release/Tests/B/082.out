32
2C 4C 5C 6C 7C QC KC AC 5D 6D 8D JD KD AD 2H 3H 4H 6H 7H 8H 9H TH JH AH 2S 3S 4S 5S 6S TS QS KS
